src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+public class Event extends Task {
+    private String start; // Start date/time
```

Comment:
Instead of comments, it may be better to consider adding information to the variable name itself eg. startTime for more readability.